force-app/main/default/classes/AccountValidation.cls

```diff
@@ -0,0 +1,38 @@
+public with sharing class AccountValidation {
+    public static void validateAccount(List<Account> accounts){
+        for(Integer i=0; i<accounts.size(); i++){
```

Comment:
busca la manera de no usar un for dentro de otro, la idea era compararlo contra base de datos